hw/dv/data/vcs/vcs.hjson

```diff
@@ -33,6 +33,57 @@
                "+UVM_TESTNAME={uvm_test}",
                "+UVM_TEST_SEQ={uvm_test_seq}"]
 
+  // Coverage related.
+  cov_db_dir: "{build_dir}/cov.vdb"
+
+  // Individual test specific coverage data - this will be deleted if the test fails
+  // so that coverage from failiing tests is not included in the final report.
+  cov_db_test_dir: "{cov_db_dir}/snps/coverage/db/testdata/{run_dir_name}"
+
+  // Merging coverage.
+  // "cov_db_dirs" is a special variable that appends all build directories in use.
+  // It is constructed by the tool itself.
+  cov_merge_dir:    "{scratch_path}/cov_merge"
```

Comment:
I will reorg our scratch output directories based on what we discussed in the next PR that will enable merging across branches. 

Also you an specify `--branch foo` to override the branch name so that the regression always goes into a specific directory regardless of the actual github branch name.

Reply:
sg, thanks.